docs/Cloud_integrations/Device_Integration_Center/overview.md

```diff
@@ -0,0 +1,88 @@
+# Data Integration Center
+
+Data Integration Center is our feature that allows you to forward device related data to your systems, databases,
```

Comment:
Data Integration Center is a feature that allows you to forward device related data to your systems, databases,
message brokers etc. There are two kind of events:

* Lifecycle - these are related to device life cycle events like creation, update, deletion
* Telemetry - these are related to telemetry gathered by device like temperature measurement